static/js/click_handlers.js

```diff
@@ -903,8 +903,7 @@ exports.initialize = function () {
     });
 
     $(".settings-header.mobile .fa-chevron-left").on("click", function () {
-        $("#settings_page").find(".right").removeClass("show");
-        $(this).parent().removeClass("slide-left");
+        window.settings_panel_menu.mobile_deactivate_section();
```

Comment:
You don't need the `window` here, and I think it's counterproductive; we the exports on `window` is the hacky temporary way to access other modules as we transition to uniformly using `require/import` to access other modules.